KidsIdKit.Web/babel.config.js

```diff
@@ -0,0 +1,4 @@
+﻿// babel.config.js
```

Comment:
I don't know the answer to this - but it seems like this file should be in the wwwroot folder?